components/brave_wallet/browser/zcash/zcash_wallet_service.cc

```diff
@@ -267,17 +270,19 @@ void ZCashWalletService::OnResolveLastBlockHeightForSendTransaction(
     SignAndPostTransactionCallback callback,
     base::expected<zcash::BlockID, std::string> result) {
   if (!result.has_value()) {
-    std::move(callback).Run("", std::move(zcash_transaction),
-                            "Couldn't obtain last block");
+    std::move(callback).Run(
+        "", std::move(zcash_transaction),
+        l10n_util::GetStringUTF8(IDS_WALLET_INTERNAL_ERROR));
```

Comment:
I know this is outside the scope of this particular change, but it seems to me that `SignAndPostTransactionCallback` should be using something like `base::expected`, rather than two separate string arguments, which come across a bit opaque. I have the impression that when the an error is encountered, the result is always `""`, and vice-versa. If so this code would be a bit easier to read if a single expected argument was used to communicate this configuration.

Reply:
Logged https://github.com/brave/brave-browser/issues/35267